### ISSUE TITLE
cpu-throttling can't be updated

### ISSUE THREAD

**Issue by csms** [googleapis/google-cloud-python#11869]
Hi - documentation says V2 should not support annotations and all changeable entities should be present in service.template - however - cpu-throttling is nowhere to be found:
https://cloud.google.com/python/docs/reference/run/latest/google.cloud.run_v2.types.RevisionTemplate
We would really appreciate to have it (use case: shut down test env's during non-office hours to save cost)

#### Environment details

  - `google-cloud-run` version: Name: google-cloud-run
Version: 0.9.1
Summary: Google Cloud Run API client library
Home-page: https://github.com/googleapis/python-run
Author: Google LLC
Author-email: googleapis-packages@google.com
License: Apache 2.0
Location: /opt/homebrew/lib/python3.11/site-packages
Requires: google-api-core, grpc-google-iam-v1, proto-plus, proto-plus, protobuf
Required-by: 

#### Steps to reproduce

  1. Run sample code

#### Code example

```python
def update_autoscaling_settings(project_id, command, locations):
    client = run_v2.ServicesClient()

    for location in locations:
        parent = f"projects/{project_id}/locations/{location}"
        try:
            services = client.list_services(parent=parent)
            for service in services:
                # Get the current service configuration
                service_config = service
                print(service)  # Print the service object to inspect its structure
                print("Current Annotations:", service.annotations)

                # Update the auto-scaling settings based on the command
                if command == "stop":
                    service_config.template.scaling.min_instance_count = 0
                    service_config.template.scaling.max_instance_count = 1
                    service_config.annotations["run.googleapis.com/cpu-throttling"] = "false"
                elif command == "start":
                    service_config.template.scaling.min_instance_count = 1
                    service_config.template.scaling.max_instance_count = 3
                    service_config.annotations["run.googleapis.com/cpu-throttling"] = "true"
                else:
                    print(f"Unknown command: {command}")


                # Create an UpdateServiceRequest object
                update_service_request = UpdateServiceRequest(
                    service=service_config,
                    validate_only=False,
                    allow_missing=False
                )

                # Use the ServicesClient to send the request
                client.update_service(request=update_service_request)
                print("Current Annotations:", service.annotations)

                return
        except Exception as e:
            print(f"Error while processing location {location}: {e}")
            return

def main(event, context):
    project = os.environ.get('PROJECT', 'Specified environment variable is not set.')
    cmd = os.environ.get('CMD', 'Specified environment variable is not set.')
    locations = os.environ.get('LOCATIONS', 'Specified environment variable is not set.').split(',')

    if 'data' in event:
        op = base64.b64decode(event['data']).decode('utf-8')
        if str(op) == cmd:
            print(project)
            update_autoscaling_settings("projects/" + project, cmd, locations)
            print("Done!")
        else:
            print("Won't run!")
            print(op)
```

#### Stack trace
```
Error while processing location europe-west2: 400 Violation in UpdateServiceRequest.service.annotations[run.googleapis.com/cpu-throttling]: system annotations are not supported in Cloud Run API v2. [field_violations {
  field: "Violation in UpdateServiceRequest.service.annotations[run.googleapis.com/cpu-throttling]"
  description: "system annotations are not supported in Cloud Run API v2."
```

Making sure to follow these steps will guarantee the quickest resolution possible.

Thanks!

**Comment by parthea** [comment]
I've created a feature request in the public issue tracker for Cloud Run [here](https://issuetracker.google.com/issues/309755065). I'm going to close this issue as a duplicate of https://issuetracker.google.com/issues/309755065.
